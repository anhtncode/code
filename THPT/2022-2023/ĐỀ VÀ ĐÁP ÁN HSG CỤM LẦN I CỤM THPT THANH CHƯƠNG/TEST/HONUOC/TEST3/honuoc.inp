5 8
707 91
268 20
726 66
883 42
261 21
