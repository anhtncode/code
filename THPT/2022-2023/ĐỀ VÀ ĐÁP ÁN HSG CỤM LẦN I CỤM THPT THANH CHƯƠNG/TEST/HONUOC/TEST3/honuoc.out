2845
2605
2365
2125
1885
1645
1405
1165
946
